server/src/service/todo.ts

```diff
@@ -0,0 +1,31 @@
+import TodoModel, { Todo } from "../models/todo";
+
+export class TodoService {
+  static async getTodoById(id: string) {
+    return TodoModel.findById(id);
+  }
+
+  static async createTodo(values: Record<string, any>) {
+    const newTodo = new TodoModel(values);
```

Comment:
I would do something like this:

const todo = await newTodo.save()
return todo.toObject()